podcasts/HowToUploadViewController.swift

```diff
@@ -67,6 +60,15 @@ class HowToUploadViewController: UIViewController {
         closeButton.accessibilityLabel = L10n.accessibilityCloseDialog
         navigationItem.leftBarButtonItem = closeButton
         navigationController?.navigationBar.setValue(true, forKey: "hidesShadow")
+
+        let image1HostVC = UIHostingController(rootView: HowToShareImage1View().environmentObject(Theme.sharedTheme))
+        guard let image1Subview = image1HostVC.view else { return }
+        addChild(image1HostVC)
+        howToShare1.addSubview(image1Subview)
+        let image2HostVC = UIHostingController(rootView: HowToShareImage2View().environmentObject(Theme.sharedTheme))
+        guard let image2Subview = image2HostVC.view else { return }
+        addChild(image2HostVC)
+        howToShare2.addSubview(image2Subview)
```

Comment:
[`didMove(toParent:`](https://developer.apple.com/documentation/uikit/uiviewcontroller/didmove(toparent:)) needs to be called here to be complete.

```suggestion
        howToShare2.addSubview(image2Subview)
        image2HostVC.didMove(toParent: self)
```